package.json

```diff
@@ -31,10 +31,11 @@
   "homepage": "https://github.com/dabapps/eslint-config-dabapps",
   "dependencies": {
     "eslint": "^6.8.0",
-    "eslint-config-prettier": "6.9.0",
+    "eslint-config-prettier": "^6.9.0",
     "eslint-import-resolver-webpack": "^0.12.1",
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-prettier": "^3.1.2",
+    "npm": "^6.13.7",
```

Comment:
This should not be installed locally.
Also, you need to discard your package-lock changes.

Reply:
oh no

Reply:
not sure how it got there - removing